frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -209,7 +211,7 @@ const Sidebar = ({
         { accounts.length ? (
           <div key="buy" className={style.sidebarItem}>
             <NavLink
-              className={({ isActive }) => isActive ? style.sidebarActive : ''}
+              className={({ isActive }) => isActive || userInSpecificAccountBuyPage ? style.sidebarActive : ''}
```

Comment:
how does this `isActive` work, where is it coming from?

Reply:
from [React Router's NavLink](https://reactrouter.com/en/main/components/nav-link). 

`isActive` is true if current route matches the `to` prop of `NavLink`